recipes/bluesky-kafka/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.4.0" %}
+{% set name = "bluesky-kafka" %}
+{% set sha256 = "55d8d2e231b851471d031a1473a0dad2f52f5cff0f6c7a7387809d9f91c911f5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky
```

Comment:
Since upstream lists `msgpack` (conda-forge: `msgpack-python`) and `msgpack-numpy` (and it imports them) it would be good to list them here rather than rely on getting them transitively.